src/webapp/ldap_data.py

```diff
@@ -165,7 +165,7 @@ def get_ligo_ldap_dns(ldap_url: str, ldapuser: str, ldap_pass: str) -> List[str]
                             queries[group],
                             search_scope='SUBTREE',
                             attributes=['gridX509subject'])
-                results += [dn for e in conn.entries for dn in e.gridX509subject]
+                results.extend(dn for e in conn.entries for dn in e.gridX509subject)
```

Comment:
Do you really think this is any more efficient though?

Reply:
`+=` makes a copy whereas `.extend()` modifies in-place, but yeah in this case the difference is negligible. Now if the _appending_ were done in a loop, that would be a different story.